src/components/Students.jsx

```diff
@@ -25,6 +37,16 @@ const Students = () => {
           console.log(err);
         });
   }, [URL]);
+  const handleClick = async (e, id) => {
+    e.preventDefault();
+    const response = deleteStudent(id);
+    console.log(response);
+    if (response) {
```

Comment:
I think you'll need to check a status code and not just the existence of a response